spec/lib/google_maps_geocoder_spec.rb

```diff
@@ -44,19 +44,19 @@
     it { should be_partial_match }
 
     context 'address' do
-      it { expect(subject.formatted_street_address).to eq '1600 Pennsylvania Avenue Northwest' }
+      it { expect(subject.formatted_street_address).to eq '1600 Pennsylvania Avenue Southeast' }
```

Comment:
Yes, this happens more frequently that you'd expect! I mean, has the White House really moved? ;)